siddhi-io/pom.xml

```diff
@@ -31,19 +31,21 @@
             <extension>
                 <groupId>org.apache.maven.archetype</groupId>
                 <artifactId>archetype-packaging</artifactId>
-                <version>3.0.1</version>
+                <version>${archetype.packaging.version}</version>
             </extension>
         </extensions>
         <pluginManagement>
             <plugins>
                 <plugin>
                     <artifactId>maven-archetype-plugin</artifactId>
-                    <version>3.0.1</version>
+                    <version>${maven.archetype.plugin.version}</version>
                     <extensions>true</extensions>
                 </plugin>
             </plugins>
         </pluginManagement>
     </build>
-
-
-</project>
\ No newline at end of file
+    <properties>
+        <archetype.packaging.version>3.0.1</archetype.packaging.version>
```

Comment:
Why do we need to properties in every pom? Can't we refer it from parent pom?

Reply:
Fixed